mantidproject/mantid
# Changes to MaxEnt

Description of work:
1. When the input is a histogram workspace MaxEnt convert X bins into data points (i.e. it extracts the middle points) to calculate the corresponding X values in the image. The image is always a "point data" workspace.
2. Adds some changes to documentation and fixes the expression for the second derivative.

**To test:**
- Check changes to documentation and new expression for entropy in Pos/Neg images
- Run one of the examples in the documentation, you should get similar plots.

Fixes #16778 
Fixes #16409

Release notes have been updated.

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
